config/default/kustomization.yaml

```diff
@@ -15,7 +15,7 @@ namePrefix: kubestellar-
 #    someName: someValue
 
 resources:
-- ../crd
+# - ../crd
```

Comment:
What was this kustomization accomplishing?

If we are no longer going to use that, perhaps it should be deleted?

Reply:
Removing this line prevents the CRDs to be injected into the chart template yaml.
When doing the same in kubefkex and otp, we commented the line instead of completely removing it.
I can remove it if you prefer

Reply:
In my own testing and analysis, it appears that this particular line was accomplishing nothing. It is a bit of generic infrastructure inserted by kubebuilder, but it never made a difference in this repo.

Removing this line makes all of config/crd/ unreferenced junk.

Reply:
ok done

Reply:
If I understand correctly, https://github.com/kubestellar/kubestellar/blob/821fb7a49c875b5aa433832b38429be457d2e86a/Makefile#L140 is also saying --- and relying on the fact that --- this repo is not using any kustomize functionality for its CRDs. @pdettori , have I got this right?

Reply:
@MikeSpreitzer correct, at least for the CRD there is currently no kustomization done.